src/stores/auth/store.ts

```diff
@@ -1,5 +1,5 @@
 import { create } from "zustand"
-import { refreshTokens, serverLogout } from "./api"
+import { refreshTokens } from "./api"
```

Comment:
Not sure if you noticed but when we come back on home screen after logging out, it still tries to refresh token which throws this error in the console as there is no token to refresh, so we should refresh token when not logged in or there is no token to refresh, you can fix this i next PR if you like to

api.ts:4 
 POST http://localhost:3000/api/auth/token 401 (Unauthorized)

Reply:
This is because the bootstrap function that runs on app mount calls refresh to attempt to get auth state. Its supposed to silently check for a token on mount to initialize state. Without it if you refreshed the page you'd lose auth state. React logs the errored fetch response even if we don't log it explicitly.